CHANGELOG.rst

```diff
@@ -0,0 +1,19 @@
+=========
+Changelog
+=========
+
+
+2.0.0 (unreleased)
+==================
+
+* Added additional settings
+* Cleaned up file structure
```

Comment:
it's kinda obvious i guess, but didn't we agree on  "BREAKING CHANGES" section of some sort?